Software/bsw/gen/CanIf_PBcfg.c

```diff
@@ -187,7 +187,7 @@ CanIfTxPduCfgType CanIfTxPduCfgObj[TX_CAN_L_PDU_NUM] =
 #if(CANIF_PUBLIC_PN_SUPPORT==STD_ON)
         TRUE,                           /* If CanIfPublicPnFilterSupport is enabled or disabled */
 #endif
-#if(CANIF_PUBLIC_READ_TX_PDU_NOTIFY_STATUS_API==STD_ON)
+#if(CANIF_PUBLIC_READTXPDU_NOTIFY_STATUS_API==STD_ON)
```

Comment:
we agreed that Macro's names are in UpperCase separated by a "_" character. so please use names in configuration files as they are.
also if you think there is anything to change in cfg files please request them in the cfg branches to be approved and pushed for the whole teams

Reply:
Because this parameter CANIF_PUBLIC_READTXPDU_NOTIFY_STATUS_API is the one used in AUTOSAR SWS as mentioned in page 54 and many other pages. Both formulas are used which is wrong and definitely creates bugs , So I decided to use only one formula